tests/unit_tests/main_test.cpp

```diff
@@ -92,7 +92,6 @@ void print_error_code(sycl::exception const& e) {
 int main(int argc, char** argv) {
     std::set<std::string> unique_devices;
     std::vector<sycl::device> local_devices;
-
```

Comment:
There are a couple of lines removed in this file which could be undone IMO.

Reply:
fixed